tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -0,0 +1,94 @@
+module.exports = {
+  commands: {
+    /**
+     *
+     * @param {string} fileName
+     */
+    openSharingDialog: function (fileName) {
```

Comment:
feels like that should stay in the filesPage Object. Because that is where you open the dialog from. The dialog does not open itself
also that method should return a sharing dialog object